server/routes/index.js

```diff
@@ -9,6 +9,7 @@ r.get('/', index);
 r.get('/healthcheck', healthcheck);
 r.get('/explore', explore);
 r.get('/articles', articles);
+r.get('/articles/(W):id', prismicArticle);
```

Comment:
We can safely assume prismic IDs start with `W`.
I am going to assume no Wordpress article do.

So if we hit a `W` => Prismic, else Wordpress.